roles/socle-config/files/crd-conf-dso.yaml

```diff
@@ -597,6 +597,10 @@ spec:
                     chartVersion:
                       description: Hashicorp Vault helm chart version (e.g., "0.25.0").
                       type: string
+                    pvcSize:
+                      description: "Size for Vault, default: 23Gi"
```

Comment:
Je pense qu'on peut mettre `Size for Vault persistent volume`, on peut avoir la valeur par défaut en faisant `kubectl describe crd/dso-socle-configs.cloud-pi-native.fr`